operator/build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/operator-framework/ansible-operator:v0.16.0
+FROM quay.io/operator-framework/ansible-operator:v0.17.0
```

Comment:
We haven't done any testing on 0.17.0. Is this required for the ARM update? Any updates to the operator SDK here will require some amount of testing (at minimum, a run of the molecule tests).

Reply:
Yes I am afraid this is needed for the arm update as v0.17.0 is the only one officially supports arm.

Reply:
OK, let's wait for this new major refactor to be complete (hopefully I will have it all merged by tomorrow afternoon US-Eastern time) and we can move over to 0.17.0 and do some testing. Unfortunately, the molecule tests are not automatically run; must be done manually. See: https://github.com/kiali/kiali/pull/2659. The move to 0.17.0 will be a separate PR in the kiali/kiali-operator repo, and the rest of this can remain, with some probable changes required once https://github.com/kiali/kiali/pull/2659 is merged - so you should probably rebase this on that PR's branch just so you can see what would need to be changed.

Reply:
Thanks very much for this detailed instruction! Take your time and I will continue my work after everything is resolved :)

Reply:
Everything is complete with the refactor. All the /operator stuff has been moved to the kiali-operator repo. See the googlegroup announcement with details on how you can configure your dev environment to work with it: https://groups.google.com/d/msg/kiali-dev/0S3-AfIW_E0/zVY0sz0WBAAJ

Reply:
This update to the operator base image of 0.17.0 is done in this PR: https://github.com/kiali/kiali-operator/pull/3

Reply:
Great to see that! I will ask my manager to extend our legal to kiali/operator repository next Monday (GMT+8 Time) and split this PR to 2 PRs. Thanks!

Reply:
Hi @jmazzitelli ! We found that there are no license files in that repo, hence we cannot apply the legal approval for it. Also even with license files, it still takes some time for processing all of them. Therefore I am wondering can you please submit code for supporting arm64 for us? just like what you did for the upgrading operator-sdk to 0.17.0? Great thanks!

Reply:
LICENSE file has been created in the repo.
I'll look at what you have done here and see if I can do it in my own PR in the operator repo.

Reply:
Great thanks!